iportal/views.py

```diff
@@ -92,7 +92,7 @@ class ApplicantOrganisationViewset(ModelViewSet):
     queryset = ApplicantOrganisation.objects.all()
     serializer_class = ApplicantOrganisationSerializer
     filter_backends = (filters.DjangoFilterBackend,)
-    filterset_fields = ('applicant','organisation_name','role')    
+    filterset_fields = ('applicant','role')    
```

Comment:
No changes in the I-Portal are required. It should be left as it is. Please restrain from making any changes to it.

Reply:
It's causing an issue, Naveen sir said that too